server/chat-plugins/mafia-data.js

```diff
@@ -359,10 +359,22 @@ const roles = {
 	role_cop: {
 		name: `Role Cop`,
 		id: `rolecop`,
-		memo: [`Role Cop: Each night you can PM the host the name of another player. You will be told their exact role, or NO RESULT if your investigation failed.`],
+		memo: [`Role Cop: Each night you can PM the host the name of another player. You will be told their exact role (not alignment), or NO RESULT if your investigation failed.`],
+		image: `<img width="75" height="75" src="//${Config.routes.client}/fx/mafia-cop.png"/>`,
+	},
+	role_cop_classic: `rolecopclassic`,
+	rolecopclassic: {
+		name: `Role Cop (Classic)`,
+		id: `rolecopclassic`,
+		memo: [`Role Cop: Each night you can PM the host the name of another player. You will be told their exact role and alignment, or NO RESULT if your investigation failed.`],
+		image: `<img width="75" height="75" src="//${Config.routes.client}/fx/mafia-cop.png"/>`,
+	},
+	vanilla_cop: {
```

Comment:
Isn't this the same as the neapolitan but under a different name?

Reply:
No, Vanilla Cop checks Vanilla / Not Vanilla, while Neapolitan specifically checks VT / not VT. It's subtle but it has very different uses in setups.